_sass/layout/_layout.scss

```diff
@@ -1,24 +1,69 @@
 // Container
-//
-// .container-left - 8/12 width container that floats left
-// .container-right - 4/12 width container that floats left
+// 
+// 
+// .container-left - 7/12 width container that floats left
+// .container-right - 5/12 width container that floats left
+// .container-left-8 - 8/12 width container that floats left
+// .container-right-4 - 4/12 width container that floats left
 // .container-half - 6/12 width container that floats left
 // .container-padded - full width container with padding
 //
 // Styleguide layout.container
 .container{
-	  @include outer-container($max-width);
+  @include outer-container($max-width);
+}
+
+%container-side {
+  @include respond-to(tiny-down) {
+    @include span-columns(12);
+    padding-left: $base-padding;
+    padding-right: $base-padding;
+    padding-bottom: $base-padding;
+  }
+  @include respond-to(tiny-up) {
+    @include span-columns(12);
+    padding-left: $base-padding;
+    padding-right: $base-padding;
+    padding-bottom: $base-padding;
+  }
+  @include respond-to(huge-up){
+    padding-left: 0;
+    padding-right: 0;
+  }
+}
+
+.container-left,
+.container-left-7 {
+  @extend %container-side;
+  @include respond-to(medium-up){
+    @include span-columns(7);
+  }
+}
+
+.container-left-8 {
+  @extend %container-side;
+  @include respond-to(medium-up){
+    @include span-columns(8);
+  }
 }
 
-.container-left {
-  @include span-columns(8);
+.container-right,
+.container-right-5 {
+  @extend %container-side;
+  @include respond-to(medium-up){
```

Comment:
@include declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space